sklearn/tests/test_calibration.py

```diff
@@ -239,6 +239,20 @@ def test_sigmoid_calibration():
                   np.vstack((exF, exF)), exY)
 
 
+def test_isotonic_calibration():
```

Comment:
you should provide a name relating to what it's testing, i.e. infinite probabilities. Are you sure that case actually happens with the test case you provided?